ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_api_gnmi.schema.yml

```
@@ -39,11 +39,70 @@ keys:
                 ip_access_group:
                   type: str
                   description: ACL name
+          grpc_tunnel:
+            type: list
+            items:
```

Add a `primary_key` so that we can check for unicity and enforce that a `name` is given for each instance
```suggestion
            type: list
            primary_key: name
            items:
```